etc/bento.env

```diff
@@ -270,12 +279,13 @@ BENTOV2_GOHAN_API_AUTHZ_REQHEADS=X-CUSTOM-1,X-CUSTOM-2
 
 # -- Elasticsearch
 BENTOV2_GOHAN_ES_USERNAME=elastic
-BENTOV2_GOHAN_ES_PASSWORD=changeme!
+# BENTOV2_GOHAN_ES_PASSWORD comes from default_config
 
 BENTOV2_GOHAN_ES_IMAGE=ghcr.io/bento-platform/gohan-elasticsearch
 BENTOV2_GOHAN_ES_VERSION=3.7.2
```

Comment:
Latest is 3.7.3, should we bump for 2.11?

Reply:
yes - will merge in latest before the final release